int8 PingScheduleRequest
---
string PingScheduleResponse
